README.md

```diff
@@ -58,3 +58,7 @@ Test
 
 You should see an HepMC printout.
 
+Documentation
+----
+
+Please consult [doc/README.md](./doc/README.md)
```

Comment:
The `doc/README.md` does not contain the links anymore, we should link to this new page, maybe:
http://fccsw.web.cern.ch/fccsw/